Refs  #21961 : Add support for database-level cascading options [Draft] (#16851)
django/django: docs/releases/5.1.txt

What's new in Django 5.1
========================

Database-level values for ``ForeignKey.on_delete``
--------------------------------------------------

> :attr:`ForeignKey.on_delete <django.db.models.ForeignKey.on_delete>` now

Reviewer: ```suggestion
:attr:`.ForeignKey.on_delete` now ...
```

What's new in Django 5.1
========================

Database-level values for ``ForeignKey.on_delete``
--------------------------------------------------

:attr:`ForeignKey.on_delete <django.db.models.ForeignKey.on_delete>` now
supports four extra values to specify database-level actions:

> * ``DB_CASCADE`` - deletes the referring object.
> * ``DB_RESTRICT`` - prevents deletion of referred-to objects.
> * ``DB_SET_NULL`` - sets the referring foreign key to SQL ``NULL``.
> * ``DB_SET_DEFAULT`` - the value of the foreign key column in the referencing
>   table will be set to its ``db_default`` value.

Reviewer 2: 1. Use imperative.
2. Shorten `DB_SET_DEFAULT` description.

```suggestion
* ``DB_CASCADE`` - delete the referring object.
* ``DB_RESTRICT`` - prevent deletion of referred-to objects.
* ``DB_SET_NULL`` - set the referring foreign key to SQL ``NULL``.
* ``DB_SET_DEFAULT`` - set the referring column to its ``db_default`` value.
```

> 
> 

Reviewer 2: Only one blank line needed.
```suggestion

```

Database-level values for ``ForeignKey.on_delete``
--------------------------------------------------

> :attr:`.ForeignKey.on_delete` now supports four extra values to specify

Reviewer 3: Was the added `.` intentional?
```suggestion
:attr:`ForeignKey.on_delete` now supports four extra values to specify
```

Author: No, sorry for the little copy paste mistake.
